File: Backend/NaissusEvents/Models/Table.cs
using System.ComponentModel.DataAnnotations;



namespace Models
{

    public class Table
    {
        [Key]
        public int Id { get; set; }

        public TableType tableType { get; set;}

        public int TableCapacity {get;set;}

        public HostingObject hostingObject {get;set;}



        

        
    }
}

File: Frontend/src/app/models/Table.ts
import { HostingObject } from "./HostingObject";

export class Table{

    id: number;
    tableType: tableType;
    tableCapacity: number;
    hostingObject: HostingObject | null;

    constructor(id: number, type: tableType,capacity:number, hostingObject:HostingObject | null) {
        this.id = id;
        this.tableType = type;
        this.tableCapacity = capacity;
        this.hostingObject = hostingObject;
    }
}

export enum tableType
{
    Barski,
    Separe,
    Niski_sto,
    Sank
}